aquilon/netbox_dump_subnetdata.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+""" netboxdump_subnetdata """
+
+import argparse
+import logging
+import os.path
+
+from json import dump
+
+import coloredlogs
+
+from scd_netbox import SCDNetbox
+
+
+class NetboxDumpSubnetdata(SCDNetbox):
+    """ Extends base SCDNetbox class with functionality to dump subnets to a file """
+
+    def _get_subnet_fields(self):
+        results = []
+        for prefix in self.netbox.ipam.prefixes.filter(tenant_id=(7, 8, 31), family=4):
```

Comment:
Hardcoded magic numbers for Tier1(ish) tenants and IPv4? Could this be ocnfigurable?

Reply:
Ooh yes! But not the IPv4 restriction as aquilon doesn't support syncing IPv6 networks.